analyzer/tests/functional/analyze_and_parse/test_analyze_and_parse.py

```diff
@@ -427,8 +427,6 @@ def test_codeclimate_export(self):
 
         self.assertEqual(result_from_stdout, result_from_file)
 
-    # TODO: Fix issue what this test catches.
-    @unittest.skip
```

Comment:
Could you please add more test cases to test this change not only for codeclimate but other output types (html,json,codeclimate,gerrit) as well?

Reply:
Done.